### PR TITLE
Fix request content type and response parsing

### PR DESCRIPTION
**Pull request by skidder**

### CODE REVIEW

File: src/main/java/com/brightcove/zencoder/client/model/State.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
     FAILED("failed"),
     CANCELLED("cancelled"),
     NO_INPUT("no_input"),
+    UPLOADING("uploading"),
```
Comment (bctvigil) [inline src/main/java/com/brightcove/zencoder/client/model/State.java]: I get the int to long changes but not sure what the context of this is.

**Comment by bctvigil** [comment]
other than comment lgtm

File: src/main/java/com/brightcove/zencoder/client/model/State.java (hunk 1 above)
Comment (skidder) [inline src/main/java/com/brightcove/zencoder/client/model/State.java]: Issue #10 prompted this change; customer reported encountering an exception while parsing responses for outputs that were in an 'uploading' state.

Comment (bctvigil) [inline src/main/java/com/brightcove/zencoder/client/model/State.java]: ok got it. lgtm

**Pull request merged**